### PR TITLE
Ignore paths

### PR DESCRIPTION
**Pull request by AdityaVallabh**
Add the list of paths that have to be skipped/ignored to a `ignorePaths.cfg` file in order to "mute" those tests

### CODE REVIEW

File: mqgo/src/meqa/mqgen/main.go [hunk 1]
```diff
@@ -29,19 +29,50 @@ func main() {
...
+			fmt.Printf("Can't load ignoredPaths file at the following location %s", pathToFile)
+			os.Exit(1)
+		}
+		paths, err := mqswag.GetListFromFile(pathToFile)
+		ignoredPaths = paths
+		if err != nil {
+			mqutil.Logger.Printf("Error: %s", err.Error())
+			os.Exit(1)
+		}
+	}
```
Comment (nitika18) [inline mqgo/src/meqa/mqgen/main.go:74]: We can move this code to a single method. The only change is the path so makes more sense to club them in a single block.

File: mqgo/src/meqa/mqgen/main.go [hunk 2]
```diff
@@ -81,7 +112,7 @@ func run(meqaPath *string, swaggerFile *string, algorithm *string, verbose *bool
 		var testPlan *mqplan.TestPlan
 		switch algo {
 		case algoPath:
-			testPlan, err = mqplan.GeneratePathTestPlan(swagger, dag)
+			testPlan, err = mqplan.GeneratePathTestPlan(swagger, dag, whitelist, ignoredPaths)
```
Comment (nitika18) [inline mqgo/src/meqa/mqgen/main.go:115]: We should use consistent names like whitelistedPaths, ignoredPaths

**Review by nitika18 — COMMENTED** [review]
This is partial review

**Comment by AdityaVallabh** [comment]
Changes made in #22

**Pull request closed without merging**